src/pages/manageUser.tsx

```diff
@@ -0,0 +1,200 @@
+            },
+            '& .MuiTabs-scroller': {
+              overflowX: 'unset !important',
+            },
+          }}
+        >
+          <Tab value={1} label={t('COMMON.FACILITATORS')} />
+        </Tabs>
+      </Box>
+      <Box>
+        {value === 1 && (
+          <>
+            <Grid
+              px={'18px'}
+              spacing={2}
+              mt={1}
+              sx={{ display: 'flex', alignItems: 'center' }}
+              container
+            >
+              <Grid item xs={8}>
+                <Box>
+                  <TextField
+                    className="input_search"
+                    placeholder="Search Facilitators.."
+                    color="secondary"
+                    focused
+                    sx={{
+                      borderRadius: '100px',
+                      height: '40px',
+                      // width: '225px',
+                    }}
+                    InputProps={{
+                      endAdornment: (
+                        <InputAdornment position="end">
+                          <SearchIcon />
+                        </InputAdornment>
+                      ),
+                    }}
+                  />
+                </Box>
+              </Grid>
+              <Grid item xs={4} marginTop={'8px'}>
+                <Box>
+                  <FormControl className="drawer-select" sx={{ width: '100%' }}>
+                    <Select
+                      className="SelectLanguages"
+                      displayEmpty
+                      style={{
+                        borderRadius: '0.5rem',
+                        color: theme.palette.warning['200'],
+                        width: '100%',
+                        marginBottom: '0rem',
+                      }}
+                    >
+                      <MenuItem className="text-4D fs-14 fw-500">
+                        {t('COMMON.FILTERS')}
+                      </MenuItem>
+                    </Select>
+                  </FormControl>
+                </Box>
+              </Grid>
+            </Grid>
+
+            <Box>
+              <Box
+                px={'18px'}
+                mt={3}
+                sx={{ borderBottom: '1px solid #D0C5B4' }}
+              >
+                <Box
+                  sx={{
+                    display: 'flex',
+                    gap: '20px',
+                    alignItems: 'center',
+                    justifyContent: 'space-between',
+                    paddingBottom: '15px',
+                  }}
+                >
+                  <Box
+                    sx={{ display: 'flex', gap: '15px', alignItems: 'center' }}
+                  >
+                    <Box>
+                      <Box
+                        sx={{
+                          fontSize: '16px',
+                          color: theme.palette.warning['300'],
+                        }}
+                      >
+                        Aditi Patel
```

Comment:
@shreyas1434shinde dont hard code values here... instead accept in a props and use